patches/minecraft/net/minecraft/client/renderer/ItemBlockRenderTypes.java.patch

```diff
@@ -74,37 +78,69 @@
 +      if (block instanceof LeavesBlock) {
 +         return f_109277_ ? type == RenderType.m_110457_() : type == RenderType.m_110451_();
 +      } else {
-+         return blockRenderChecks.get(block.delegate).test(type);
++         return getBlockLayerPredicates().get(block.delegate).test(type);
 +      }
 +   }
 +
 +   public static boolean canRenderInLayer(FluidState fluid, RenderType type) {
-+      return fluidRenderChecks.get(fluid.m_76152_().delegate).test(type);
++      return getFluidLayerPredicates().get(fluid.m_76152_().delegate).test(type);
 +   }
 +
 +   public static void setRenderLayer(Block block, RenderType type) {
 +      setRenderLayer(block, createMatchingLayerPredicate(type));
 +   }
 +
 +   public static synchronized void setRenderLayer(Block block, java.util.function.Predicate<RenderType> predicate) {
```

Comment:
Very tiny thing- maybe makes sense to remove the `synchronized` modifier now?

Reply:
Yeah, that can be safely removed, I'll do that real quick